### DISCUSSION TITLE
Base operating system

### DISCUSSION THREAD

**Discussion by abriko** [routernetes/routernetes#10] [General]
Hi

Is possible choose the base operating system?

CoreOS is awesome, but what if I want to use Debian or other Linux as base operating system.

Do we have guide setup routernetes without system.

**Comment by loganmc10** [comment]
It's possible, and actually, the CI that runs for the Operator deploys it on Ubuntu, see https://github.com/routernetes/routernetes-operator/blob/main/.github/workflows/test-operator.yaml

The problem is mostly support, if more people got involved in the development, then yes. As just 1 person, I can't really commit to supporting multiple configurations.

The Operator does make certain assumptions about the underlying OS, specifically, it requires NetworkManager and firewalld to be installed, so it's not something that would work out-of-the-box on every OS.
